hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/ContractResultServiceImpl.java

```diff
@@ -249,6 +252,21 @@ private void processContractResult(
         entityListener.onContractResult(contractResult);
     }
 
+    private void processContractNonce(ContractFunctionResult functionResult) {
+        var nonceInfos = functionResult.getContractNoncesList();
+        if (!nonceInfos.isEmpty()) {
+            nonceInfos.forEach(nonceInfo -> {
+                var contractId =
+                        entityIdService.lookup(nonceInfo.getContractId()).orElse(EntityId.EMPTY);
```

Comment:
We don't need to use the entity service here as entity IDs in the record are guaranteed to always be in shard.realm.num format.

```suggestion
                var contractId = EntityId.of(nonceInfo.getContractId());
```